common/app/experiments/Experiments.scala

```diff
@@ -37,3 +37,12 @@ object OfferHttp3
       sellByDate = LocalDate.of(2022, 12, 6),
       participationGroup = Perc0B,
     )
+
+object KeyEventsCarousel
+    extends Experiment(
+      name = "key-events-carousel",
+      description = "Displays key events inside a carousel",
+      owners = Seq(Owner.withGithub("abeddow91")),
+      sellByDate = LocalDate.of(2022, 7, 13),
```

Comment:
This only gives us 30 days to test with the opt in - very unlikely that we need more but on the off chance should we give ourselves another couple of months?

Reply:
sure